deploy/harvester-cloud-provider/templates/deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   {{- include "harvester-cloud-provider.labels" . | nindent 4 }}
   name: {{ include "harvester-cloud-provider.name" . }}
 spec:
-  replicas: {{ .Values.replicaCount}}
+  replicas: {{ .Values.replicasCount}}
```

Comment:
the `replicaCount` spelling is correct, but u are missing space at the end of `replicaCount }}`